tokio/src/runtime/blocking/shutdown.rs

```diff
@@ -26,7 +28,7 @@ pub(super) fn channel() -> (Sender, Receiver) {
 
 impl Receiver {
     /// Blocks the current thread until all `Sender` handles drop.
-    pub(crate) fn wait(&mut self) {
+    pub(crate) fn wait(&mut self, timeout: Option<Duration>) {
```

Comment:
Nit: could be helpful to add a note that if `timeout` is `None`, this waits forever in the doc comment above? Not a blocker as this is an internal API...